### ISSUE TITLE
Project still live???

### ISSUE THREAD

**Issue by luizfelipers19** [Coswold/Dragon_Ball_API#1]
Hello Connor, how are you?
I'm opening this issue to ask you if the API still live, because I'm trying to access data from the URL on the browser and it returns me errors. I tried it with the characters and planets URL and didn't manage to fetch the data. =(

Thanks for your attention!

**Comment by castelles** [comment]
Had the same problem. API seems off.

**Comment by jakesmith4** [comment]
Yes, me too, please fix it asap, This is a really cool API and I really want to connect to it. Please comment back or keep us updated on the statis of this. Thanks!!

**Comment by chadvidovcich** [comment]
@Coswold  are you willing to transfer ownership of this project to me? If you have the MongoDB set up, I can transfer the data to mine. I and a lot of others would like to see this project active. 😁
